lms/templates/courseware/courseware.html

```diff
@@ -181,21 +181,29 @@
                                 <span class="nav-item nav-item-course">
                                     <a href="${course_url}">${course_home_page_title(course)}</a>
                                 </span>
-                                <span class="icon fa fa-angle-right" aria-hidden="true"></span>
+                                % if chapter or section or sequence_title:
+                                    <span class="icon fa fa-angle-right" aria-hidden="true"></span>
+                                % endif
```

Comment:
@mbasaglia, what's the point of having these checks? Why don't we just move these arrows to the next items, like I've suggested to you before?
> We'll also need to move the `fa-angle-right` arrows to be displayed before the text to avoid having a trailing arrow when the unit's name is unavailable.

Instead of duplicating conditions, we could simply do something like:
```html
% if sequence_title:
    <span class="icon fa fa-angle-right" aria-hidden="true"></span>
    <span class="nav-item nav-item-sequence">${sequence_title}</span>
% endif</span>
```

Reply:
it's to also avoid extra leading icons, since you can't assume you have a previous item

Reply:
@mbasaglia, in this case, we can completely remove these `span`s and rely on the CSS instead of adding extra logic to templates.
We can remove [this](https://github.com/edx/edx-platform/blob/master/common/static/sass/edx-pattern-library-shims/_breadcrumbs.scss#L46-L60) and add the following as the last rule of the `.nav-item` class:
```css
&:not(:last-child)::after {
    content: '\f105';
    font-family: FontAwesome;

    @include margin-left($baseline/4);

    display: inline-block;
    color: $body-color;

    @include rtl {
      @include transform(rotateY(180deg));
    }
}
```
Note: I've intentionally skipped `media-breakpoint-down` here, as it's broken on mobile resolutions.